src/main/java/org/sagebionetworks/bridge/exporter/handler/SynapseExportHandler.java

```diff
@@ -47,8 +46,13 @@
     private static final Logger LOG = LoggerFactory.getLogger(SynapseExportHandler.class);
 
     // Package-scoped to be available to unit tests.
-    static final List<ColumnModel> COMMON_COLUMN_LIST;
-    static {
+    private List<ColumnModel> commonColumnList;
+
+    private List<ColumnDefinition> columnDefinition;
+
+    private void setSynapseColumnDefinitionsAndColumnList() {
```

Comment:
Minor: Rename this to something like "initSynapseColumnDefinitionsAndColumnList", so we don't get it confused with an actual setter.